documentation/tutorials/Demo_02_Uncertainty_Analysis/uncertainty.qmd

```diff
@@ -0,0 +1,597 @@
+    ```
+
+# 7. Run Model Simulations
+
+This section executes the SIPNET simulations and retrieves the results.
+
+It uses the function `runModule_start_model_runs(settings)` to initiate the model runs using the configuration files generated in the previous step.
+
+```{r run-model}
+PEcAn.workflow::runModule_start_model_runs(settings)
+```
+
+The PEcAn workflow will take a longer time to complete than in Demo 1 because we have just asked for over a hundred model runs. Once the runs are complete we will continue.
+
+# 8. Fetch Model Outputs
+
+Next we convert all of the model output from the previous run to a standard format that PEcAn can use for analysis. This is done using the `runModule.get.results()` function.
+
+```{r get-model-results}
+runModule.get.results(settings)
+```
+
+# 8. Ensemble and Sensitivity Analysis
+
+Next, we use the outputs from the previous step to perform ensemble and sensitivity analyses.
+
+**Ensemble Analysis**: Quantifies uncertainty in model predictions by running multiple simulations with parameters sampled from their uncertainty distributions. This generates probability distributions of model outputs and confidence intervals.
+
+```{r run-ensemble-analysis}
+runModule.run.ensemble.analysis(settings)
+```
+
+**Sensitivity Analysis**: Systematically varies individual parameters to assess their influence on model outputs. This identifies which parameters most strongly affect model predictions and helps prioritize parameter refinement efforts.
+
+```{r run-sensitivity-analysis}
+runModule.run.sensitivity.analysis(settings)
+```
+
+# 9. Output 
+
+## Output organization
+
+These are the key folders and files that will be created under the directory defined by `settings$outdir` (e.g., `demo_outdir` in the example). The file contents are described in the next section.
+
+We have discussed the structure of the output directory in Demo 1 (Section 10), but now we have three new folders that contain outputs from the sensitivity, ensemble, and variance decomposition analyses. Here we focus on the additional outputs generated by the ensemble and sensitivity analyses.
+
+```
+demo_outdir/
+├── run/                   # Configuration & execution metadata
+│   ├── runs.txt           # List of run IDs (one per model realization)
+│   ├── <runid>/           # Model-specific config copies (sometimes)
+│   └── config.*           # Generated model configs (e.g., SIPNET)
+├── out/                   # Raw model outputs by run ID
+│   └── <runid>/           # E.g., daily or sub-daily SIPNET output files
+├── sensitivity.analysis/  # Sensitivity raw outputs &amp; figures
+│   ├── sensitivity.output.*.Rdata
+│   └── sensitivity.analysis.*.pdf
+├── ensemble.analysis/     # Ensemble summaries &amp; figures
+│   ├── ensemble.Rdata
+│   ├── ensemble.analysis.*.pdf
+│   └── ensemble.ts.*.pdf
+├── variance.decomposition/  # Created when variance decomposition runs
+│   └── variance.decomposition.*.pdf
```

Comment:
I'm used to seeing all of these saved directly in outdir without their own subdirectories -- what process creates these?